build-system/tasks/performance/helpers.js

```diff
@@ -13,100 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-const fetch = require('node-fetch');
-const fs = require('fs');
 const path = require('path');
-
-const CDN_URL = 'https://cdn.ampproject.org/';
 const CONTROL = 'control';
 const EXPERIMENT = 'experiment';
-const CACHE_PATH = path.join(__dirname, './cache');
-const CONTROL_CACHE_PATH = path.join(CACHE_PATH, `./${CONTROL}`);
-const EXPERIMENT_CACHE_PATH = path.join(CACHE_PATH, `./${EXPERIMENT}`);
+const HOST = 'localhost';
```

Comment:
This value is variable based on configuration for the local server.

Reply:
?? This is passed as the local server config.

Reply:
The default `gulp` task and `gulp serve` can accept an argument 'host' which changes the local server options.

```javascript
const options = {
    name: 'AMP Dev Server',
    root: process.cwd(),
    host: argv.host || 'localhost',
    port: argv.port || 8000,
    https: argv.https,
    preferHttp1: true,
    silent: true,
    middleware: getMiddleware,
    ...connectOptions,
  };
```

Does this change account for a host and port being specified?

Reply:
Ah I see what you mean. It shouldn't matter, because they are passed as `connectOptions` which overwrite the initial values.

Reply:
Awesome, thanks!